src/RegistrationInterface.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Firehed\U2F;
+
+use Serializable;
```

Comment:
Left over from something? I don't see it being used.

Reply:
Oops! Removed.